proto/proxy/telemetry/telemetry.proto

```diff
@@ -67,21 +67,12 @@ message RequestCtx {
   string path = 5;
 }
 
-// A latency value in tenths of a millisecond and a count of the times
-// that latency was observed.
-message Latency {
-    // Latency value in tenths of a millisecond.
-    uint32 latency = 1;
-    // Count of occurences of this latency value.
-    uint32 count = 2;
-}
-
 message ResponseScope {
   ResponseCtx ctx = 1;
   // Response latencies (time from request headers sent to response headers received),
```

Comment:
@olix0r how do you feel about using pre-known buckets and not encoding the bucket bounds in the proto message at all?  too inflexible, or okay?

Reply:
After thinking about this more, I think we do need to include the bucket bounds somewhere in the message so that the proxy and controller can tolerate a version mismatch where the bucket bounds change.  But the bounds need only be sent once per report instead of once per histogram.

Reply:
this comment (and potentially the field name) need to be updated to indicate that this is an array of counts.  The array must be the same length as the bucket bounds array and each value represents the number of observations in the corresponding bucket.